static/fetchMovieScreenings.js

```diff
@@ -0,0 +1,8 @@
+export default async function fetchMovieScreenings() {
+    const currentPath = window.location.pathname
+    const base = window.location.origin;
+    const url = new URL("/api" + currentPath + "/screenings", base)
```

Comment:
There are some neat solutions here, but also a bit unusual, so here are some comments on alternative solutions.

First of all, I think it's neat that you're retrieving the ID of the current movie from the path. The problem is that you're not just getting the ID, but also the full path. That means that you're effectively tying your API URLs and your website/frontend URLs to each other, so you're missing out on separating those concerns from each other. If you change one, you will have to change the other (or this code).

So it would typically be better to get the ID from the movie from somewhere else, e.g. from a parameter passed in from HTML rendered by your handlebars template, by calling `fetchMovieScreenings({{{ movie.id }}})` in a `<script>` tag inside your handlebars code. This can be confusing, because on the backend it looks like a weird mix of JS and handlebars, but the important thing is what it looks like on the frontend, where it will just be `fetchMovieScreenings(1)` for movie 1, and `fetchMovieScreenings(2)` etc.